website/src/components/CustomTag/index.tsx

```diff
@@ -11,78 +9,25 @@ interface blogTitleProps {
 const CustomTag: React.FC<blogTitleProps> = ({ blogLabel }) => {
   const classes = useStyles();
 
-  const chaosEngineering = () => {
-    return (
-      <Typography variant="h6" className={classes.chaosengineering}>
+  const getBackgroundColor = () => {
+    const backgroundColorsDictionary = ['#FFE2D5', '#FFEDAD', '#DBF0F7', '#62A7BD', '#464497', '#DCDBF7', '#B2EFE8', '#FFCDCD'];
+    const backgroundColor = backgroundColorsDictionary[Math.floor(Math.random() * backgroundColorsDictionary.length)];
+    return backgroundColor;
+  }
+
+  const getTextColor = () => {
+    const textColorsDictionary = ['#F26D00','#073D47','#FFFFFF', '#A4451B', '#3B4473', '#3B4473', '#F26D00', '#A4451B', '#A4451B', '#6A5711', '#62A7BD', '#223288', '#3D9086', '#963D16'];
+    const textColor = textColorsDictionary[Math.floor(Math.random() * textColorsDictionary.length)];
+    return textColor;
+  }
+
+  return (
+    <>
+       <Typography variant="h6" className={classes.tag} style = {{backgroundColor: getBackgroundColor(), color: getTextColor()}}>
```

Comment:
Can we form an object array instead which will set the correct text color for the randomly picked background color?

```suggestion
const getTabStyle = () => {
  const colorsDictionary = [
    {
    'background': '#FFE2D5',
    'text': '#A4451B'
    },
    {
      'background': '#FFEDAD',
      'text': '#6A5711'
    },
    {
      'background': '#DBF0F7',
      'text': '#62A7BD'
    },
    {
      'background': '#DCDBF7',
      'text': '#223288'
    },
    {
      'background': '#B2EFE8',
      'text': '#3D9086'
    },
    {
      'background': '#FFCDCD',
      'text': '#963D16'
    }
];
  const randomColor = colorsDictionary[Math.floor(Math.random() * colorsDictionary.length)];
  const tabStyle = {backgroundColor: randomColor.background, color: randomColor.text}
  return tabStyle;
};


  return (
    <>
       <Typography variant="h6" className={classes.tag} style = {getTabStyle()}></Typography>
```

Because with the current method some time we get non-contrasting colors for background and color since we are generating both random color and background separately.

Here is a example, (text over blue tab is not visible).
<img width="1439" alt="Screenshot 2021-06-09 at 7 04 12 PM" src="https://user-images.githubusercontent.com/34312966/121467567-e65d7180-c9d6-11eb-8b33-9347e2313efb.png">

Also, this ensures we have color combinations for the background as given in Figma screen

<img width="446" alt="Screenshot 2021-06-10 at 10 32 32 AM" src="https://user-images.githubusercontent.com/34312966/121467703-30deee00-c9d7-11eb-9c03-57d3447e46d7.png">

Reply:
Done.